src/org/sparkboard/slack/screens.clj

```diff
@@ -0,0 +1,294 @@
+                                reply-ref (-> broadcast-ref (.child "replies") (.push))]
+                            (fire-jvm/set-value reply-ref {:from-channel-id (-> payload :channel :id)})
+                            (v/open! team-broadcast-response-compose
+                                     (assoc context
+                                       :initial-state {:original-message original-message
+                                                       :broadcast/path (fire-jvm/ref-path broadcast-ref)
+                                                       :broadcast/reply-path (fire-jvm/ref-path reply-ref)
+                                                       :broadcast/reply-ts (-> payload :message :ts)
+                                                       :broadcast/reply-channel (-> payload :channel :id)}))))}}
+              "Reply"]]
+            [:context
+             [:md "Replies will be sent to " (v/channel-link response-channel)]]))))
+
+(defn send-broadcast! [context {:as opts
+                                :keys [message
+                                       response-channel
+                                       collect-in-thread]}]
+  (let [broadcast-ref (.push (fire-jvm/->ref "/slack-broadcast"))
+        sender-name (-> context :slack/payload :user :name)
+        {thread :ts} (when response-channel
+                       (slack/web-api "chat.postMessage"
+                                      {:channel response-channel
+                                       :blocks [[:section
+                                                 [:md
+                                                  "*" sender-name "*"
+                                                  " sent a message to all teams:\n"
+                                                  (v/blockquote message)]]]}))
+        content (if response-channel
+                  {:blocks (team-broadcast-content (merge context
+                                                          #:broadcast{:id (.getKey broadcast-ref)
+                                                                      :message message
+                                                                      :response-channel response-channel}))}
+                  {:text (v/blockquote (:message opts))})]
+    (fire-jvm/set-value broadcast-ref
+                        {:message message
+                         :response-channel response-channel
+                         :response-thread (when collect-in-thread thread)})
+    (mapv #(slack/web-api "chat.postMessage"
+                          {:auth/token (:slack/bot-token context)}
+                          (merge
+                            content
+                            {:channel (:channel-id %)
+                             :unfurl_media true
+                             :unfurl_links true}))
+          (slack-db/team->all-linked-channels (:slack/team-id context)))))
+
+(v/defview team-broadcast-compose
```

Comment:
We should make sure to support docstrings in `defview`, so that in a future PR we can add docstrings to these.

Reply:
Yup, done